servers/republik/modules/crowdfundings/lib/scheduler/owners.js

```diff
@@ -165,10 +167,11 @@ const inform = async (args, context) => {
         const templatePayload = await context.mail.prepareMembershipOwnerNotice({
           user,
           endDate: prolongBeforeDate,
-          cancelUntilDate: moment(prolongBeforeDate)
-            .subtract(2, 'days'),
+          cancelUntilDate: moment(prolongBeforeDate).subtract(2, 'days'),
```

Comment:
`cancelUntilDate` will always be in the past for `membership_owner_prolong_notice_0` right?

Reply:
Yes.